aiida/engine/processes/calcjobs/calcjob.py

```diff
@@ -429,11 +429,14 @@ def _perform_import(self):
         from aiida.common.datastructures import CalcJobState
         from aiida.common.folders import SandboxFolder
         from aiida.engine.daemon.execmanager import retrieve_calculation
+        from aiida.manage import get_config_option
```

Comment:
A bit of a nitpick, but do we have any guidelines as to how to import features that are exposed at higher levels? Here you are importing `get_config_option` from `aiida.manage`, but previously you did it from the more specific `aiida.manage.configuration`.

Reply:
True, there aren't any guidelines. I think there is something to be said for both, but I also don't think it matters that much. Feel free to open a discussion if you think it would be good to have a guideline for this.